src/extension/background-script/connectors/commando.ts

```diff
@@ -142,7 +142,15 @@ export default class Commando implements Connector {
   }
 
   get supportedMethods() {
-    return supportedMethods;
+    return [
+      "getInfo",
+      "keysend",
+      "makeInvoice",
+      "sendPayment",
+      "signMessage",
+      "getBalance",
+      ...supportedMethods.map((i) => `request.${i}`),
```

Comment:
there are some inconsistencies here (2 other places it is `...Object.keys(methods).map((i) => `request.${i}`),`)

I wonder if we should call them all "weblnRequestSupportedMethods" or something to make it specific, then have a helper method to make sure they all use the same flattening function

Reply:
Good point. I added a `flattenRequestsMethods()` function that is used in all connectors.